sklearn/neighbors/_kde.py

```diff
@@ -31,43 +31,41 @@ class KernelDensity(BaseEstimator):
 
     Parameters
     ----------
-    bandwidth : float
+    bandwidth : float, default=1.0
         The bandwidth of the kernel.
 
-    algorithm : str
-        The tree algorithm to use.  Valid options are
-        ['kd_tree'|'ball_tree'|'auto'].  Default is 'auto'.
+    algorithm : {'kd_tree', 'ball_tree', 'auto'}, default='auto'
+        The tree algorithm to use.
 
-    kernel : str
-        The kernel to use.  Valid kernels are
-        ['gaussian'|'tophat'|'epanechnikov'|'exponential'|'linear'|'cosine']
-        Default is 'gaussian'.
+    kernel : {'gaussian', 'tophat', 'epanechnikov', 'exponential', 'linear', \
+                 'cosine'}, default='gaussian'
```

Comment:
Nit:

```suggestion
    kernel : {'gaussian', 'tophat', 'epanechnikov', 'exponential', 'linear', \
              'cosine'}, default='gaussian'
```

Reply:
Uhm I am not sure because it is intended with the below paragraph. Usually, you don't want that.